--- a/device/google/crosshatch/aosp_crosshatch.mk	2020-01-04 11:42:41.045071657 +0000
+++ b/device/google/crosshatch/aosp_crosshatch.mk	2020-01-04 11:44:12.564349001 +0000
@@ -22,6 +22,7 @@
 $(call inherit-product-if-exists, vendor/google_devices/crosshatch/proprietary/device-vendor.mk)
 
 PRODUCT_COPY_FILES += $(LOCAL_PATH)/audio_effects.xml:$(TARGET_COPY_OUT_VENDOR)/etc/audio_effects.xml
+PRODUCT_RESTRICT_VENDOR_FILES := false
 
 # STOPSHIP deal with Qualcomm stuff later
 # PRODUCT_RESTRICT_VENDOR_FILES := all
--- a/device/google/crosshatch/device-crosshatch.mk	2020-01-04 11:42:41.097071133 +0000
+++ b/device/google/crosshatch/device-crosshatch.mk.2	2020-01-04 11:44:47.964147383 +0000
@@ -14,6 +14,7 @@
 # limitations under the License.
 #
 
+GAPPS_VARIANT := pico
 PRODUCT_HARDWARE := crosshatch
 
 include device/google/crosshatch/device-common.mk
@@ -34,3 +35,4 @@
 
 PRODUCT_PACKAGES += \
     NoCutoutOverlay
+$(call inherit-product, vendor/opengapps/build/opengapps-packages.mk)
