--- a/vendor/opengapps/build/core/find_apk.sh	2019-10-28 11:28:28.079239000 -0600
+++ b/vendor/opengapps/build/core/find_apk.sh	2019-10-28 11:32:52.957941115 -0600
@@ -15,8 +15,8 @@
   local LZ_DECOMPRESS_ARG=""
   if command -v lunzip > /dev/null; then
     LZ_DECOMPRESS_CMD="lunzip"
-  elif command -v lzip > /dev/null; then
-    LZ_DECOMPRESS_CMD="lzip"
+  elif command -v /usr/bin/lzip > /dev/null; then
+    LZ_DECOMPRESS_CMD="/usr/bin/lzip"
     LZ_DECOMPRESS_ARG="--decompress"
   fi;
