--- a/vendor/opengapps/build/core/prebuilt_apk.mk
+++ b/vendor/opengapps/build/core/prebuilt_apk.mk
@@ -42,6 +42,10 @@ else
       ifdef LOCAL_SRC_FILES
         LOCAL_MODULE_TARGET_ARCH := $(TARGET_2ND_ARCH)
         LOCAL_PREBUILT_JNI_LIBS_$(TARGET_2ND_ARCH) := $(call find-libs-in-apk,$(TARGET_2ND_ARCH),$(LOCAL_SRC_FILES))
+      else
+        LOCAL_SRC_FILES := $(call find-apk-for-pkg,arm,$(LOCAL_PACKAGE_NAME))
+        LOCAL_MODULE_TARGET_ARCH := arm
+        LOCAL_PREBUILT_JNI_LIBS_arm := $(call find-libs-in-apk,$(TARGET_2ND_ARCH),$(LOCAL_SRC_FILES))
       endif
     endif
   endif
-- 
2.23.0
